pippo-core/src/main/java/ro/pippo/core/route/Route.java

```diff
@@ -117,13 +119,28 @@ public String getRequestMethod() {
     }
 
     public String getUriPattern() {
-        return uriPattern;
+//        return uriPattern;
+        return getCompletePath();   //should i modify this method or modify methods in DefaultRouter to use getCompletePath ?
     }
 
     public RouteHandler getRouteHandler() {
         return routeHandler;
     }
 
+    // path with group
+    public String getCompletePath() {
```

Comment:
Maybe better is `getFullUriPattern`. 
Other possible idea is:
- move content of getComplete in getUriPattern
- remove getComplete
- create a method getRelativeUriPattern that return uriPattern variable